magicctapipe/scripts/lst1_magic/semi_automatic_scripts/DL1_to_DL2.py

```diff
@@ -4,7 +4,7 @@
 the data level 2.
 
 Usage:
-$ DL1_to_DL2 -c configuration_file.yaml
+$ DL1_to_DL2 -c configuration_file.yaml -d list_dense.txt
```

Comment:
this -d option is optional as far as I can see (and should be) so you can put it here as [-d list_dense.txt]